www/README.md

```
@@ -0,0 +1,41 @@
+# Website
+
+This website is built using [Docusaurus](https://docusaurus.io/), a modern static website generator.
+
+### Installation
+
+```
+$ yarn
```

Markdownlint (0.37.0)

Dollar signs used before commands without showing output

(MD014, commands-show-output)